static/index.html

```diff
@@ -1,332 +1,520 @@
+   <!-- block-one -->
+  <div class="w-full bg-one bg-white-lilac">
+    <div class="grid grid-cols-1">
+      <h2 class="pt-54 mb-4 text-center">
+        CONNECTING ALL<br class="block md:hidden"> THE BLOCKCHAINS<br class="hidden md:block">
+        WITHOUT<br class="block md:hidden"> ADDING YET ANOTHER ONE
+      </h2>
+      <p class="text-center px-3">
+        We believe that a good cross-blockchain solution is one<br class="block md:hidden"> 
+        that minimizes complexity instead of adding<br class="hidden md:block">
+        to it.<br class="block md:hidden"> The COMIT open source protocol is designed to be:
+      </p>
+      <div class="md:m-auto md:max-w-xxl md:pr-16 grid grid-cols-12">
+          <p class="font-bold text-klein-blue pt-28 md:pt-9 text-right col-span-4 md:col-span-5">
+            PURE &nbsp;&mdash;&nbsp;
+          </p>
+          <p class="pt-28 md:pt-9 col-span-8 md:col-span-7 pl-1 whitespace-no-wrap">          
+            No additional chain. No additional token
+          </p>
+          <p class="font-bold text-klein-blue pt-28 md:pt-9 text-right col-span-4 md:col-span-5">
+            NATIVE &nbsp;&mdash;&nbsp;
+          </p>
+          <p class="pt-28 md:pt-9 col-span-8 md:col-span-7 pl-1">          
+            Any blockchain. Any layer
+          </p>
+          <p class="font-bold text-klein-blue pt-28 md:pt-9 text-right col-span-4 md:col-span-5">
+            TRUSTLESS &nbsp;&mdash;&nbsp;
+          </p>
+          <p class="pt-28 md:pt-9 col-span-8 md:col-span-7 pl-1">          
+            No trusted 3rd parties. Verify, don’t trust
+          </p>
+          <p class="font-bold text-klein-blue pt-28 md:pt-9 text-right col-span-4 md:col-span-5">
+            ACCOUNTLESS &nbsp;&mdash;&nbsp;
+          </p>
+          <p class="pt-28 md:pt-9 col-span-8 md:col-span-7 pl-1">          
+            No need to create an account or deposit tokens
+          </p>
+          <p class="font-bold text-klein-blue pt-28 md:pt-9 text-right col-span-4 md:col-span-5">
+            OPEN &nbsp;&mdash;&nbsp;
+          </p>
+          <p class="pt-28 md:pt-9 col-span-8 md:col-span-7 pl-1">          
+            Tools and codebase that are open source
+          </p>
+      </div>
+    </div>
+  </div>
+  <!-- /block-one -->
+  <!-- .block-two -->
+  <div class="px-24  w-full bg-two">
+    <h2 class="pt-84 md:pt-18 mb-4 text-center">THE TECH BEHIND THE SWAP</h2>
+    <div class="grid grid-cols-2 pt-50 md:pt-13">
+      <div class="pr-40 md:pr-10 my-auto">
+        <img class="float-right w-92 md:w-33 md:pt-20" src="img/case.svg" alt="OPEN DEVELOPER CENTRIC TOOLS">
+      </div>
+      <div class="text-left">
+        <h3>OPEN DEVELOPER <br>CENTRIC TOOLS</h3>
+        <p class="md:pt-4 ">
+          Use <strong>create-comit-app</strong> to spin-up a<br class="hidden md:block"> 
+          development environment. The Javascript SDK<br class="hidden md:block">
+          allows simple usage of COMIT’s powerful API.
```

Comment:
Do we want this bold here? If so, we should use it consistently, e.g. 

```suggestion
          Use <strong>create-comit-app</strong> to spin-up a<br class="hidden md:block"> 
          development environment. The <strong>Javascript SDK</strong><br class="hidden md:block">
          allows simple usage of COMIT’s powerful API.
```